### PR TITLE
OCF Host Onboarding Flow: Registered user with collective implementation

### PR DESCRIPTION
**Pull request by flickz**
Following up https://github.com/opencollective/opencollective-frontend/pull/5623 on the recent OCF onboarding flow [update](https://github.com/opencollective/opencollective/issues/3683#issuecomment-769566325). 

### Screen record
https://user-images.githubusercontent.com/15707013/106905291-6d829c00-66fc-11eb-9ba1-3abaad0ebcef.mov

### CODE REVIEW

**Comment by znarf** [comment]
@flickz there are huge conflicts, we merged a 220 files PR this morning touching a bit everything. I can help resolving them if you wish.

At the end of the screen-record, it's surfaced that the Collective was already Hosted by the OCF. Do we want to mitigate this problem?

**Comment by flickz** [comment]
>there are huge conflicts, we merged a 220 files PR this morning touching a bit everything. I can help resolving them if you wish.

@znarf  Please do, thanks. 

> At the end of the screen-record, it's surfaced that the Collective was already Hosted by the OCF. Do we want to mitigate this problem?

Yeah, I fixed that after recording.

**Comment by znarf** [comment]
I forced push with conflicts resolved.

File: components/ApplyToHostModal.js [hunk 1]
```diff
@@ -310,7 +339,10 @@ const ApplyToHostModal = ({ hostSlug, collective, onClose, onSuccess, ...props }
                               types={['COLLECTIVE']}
                               creatable
                               renderNewCollectiveOption={() => (
-                                <Link route={`/${host.slug}/apply`} data-cy="host-apply-new-collective-link">
+                                <Link
+                                  route={isOCFHost ? `/foundation/apply/intro` : `/${host.slug}/apply`}
```
Comment (Betree) [inline components/ApplyToHostModal.js:343]: https://opencollective.com/foundation/apply does the same effect no?

**Pull request merged**